src/site/shared/containers/ContextMenu/index.tsx

```diff
@@ -118,33 +140,30 @@ export const ContextMenu = ({info, paste}: Props) => {
         dispatch(CloseContextMenu());
     }
 
+
     const menu = useRef<HTMLDivElement>(null);
-    let pos = input?.getMousePos();
 
-    /* Relocate context menu to opposite side of cursor if it were to go off-screen */
-    if (isOpen) {
-        if (!menu.current)
-            throw new Error("ContextMenu failed: menu.current is null");
+    // Adjusts position of menu to keep it on screen
+    let pos = V(posX, posY);
+    if(menu.current){
         const offset = 1;
         const contextMenuWidth = menu.current.getBoundingClientRect().width;
         const contextMenuHeight = menu.current.getBoundingClientRect().height;
```

Comment:
I know you didn't write this, but we can simplify this to:
```ts
const {width, height} = menu.current.getBoundingClientRect();
```
i believe (and then change the if-statements below to use `width`,`height` instead